test/web/controllers/login_controller_test.exs

```diff
@@ -2,21 +2,21 @@ defmodule Advisor.Web.LoginControllerTest do
   use Advisor.Web.ConnCase
 
   test "Proper login to request page", %{conn: conn} do
-    conn = post conn, "/begin", [email: "felipe@example.com", request: ""]
+    conn = post conn, "/begin", [email: "felipe@example.com", submit: "advice"]
 
     assert redirected_to(conn) =~ "/request"
     assert conn.cookies["user"] == "11"
   end
 
   test "Login to the dashboard", %{conn: conn} do
-    conn = post conn, "/begin", [email: "felipe@example.com", dashboard: ""]
+    conn = post conn, "/begin", [email: "felipe@example.com", submit: "dashboard"]
 
     assert redirected_to(conn) =~ "/dashboard"
     assert conn.cookies["user"] == "11"
   end
 
   test "Fails to login with unkown email", %{conn: conn} do
-    conn = post conn, "/begin", [email: "bob@example.com", request: ""]
+    conn = post conn, "/begin", [email: "bob@example.com", submit: "advice"]
```

Comment:
Not sure...  I've coded those values in the login form. 
I needed a common name amongst all buttons (submit) and a differentiator to distinguish the intent of the user (advice, dashboard, redirect). 

Do you have any alternatives in mind?